omniledger/service/service.go

```diff
@@ -396,6 +400,18 @@ func (s *Service) updateCollection(msg network.Message) {
 		log.Error("Couldn't recreate state changes:", err.Error())
 		return
 	}
+
+	// Check whether we are storing a genesis config, if we are, also set
+	// the leaderMap.
+	if scs.isGenesisConfig() {
```

Comment:
Why is that here and not in the `Config`-contract?

Reply:
probably will be removed if `leaderMap` is removed